tests/api.js

```diff
@@ -70,6 +73,10 @@ describe('Initialization', function() {
   });
 
   it('should detect missing cloudant', function(done) {
+    if (process.env.NOCK_OFF) {
+      this.skip();
```

Comment:
Worth checking there aren't any `afterEach` in places where `skip` is used because mocha doesn't run them see https://github.com/mochajs/mocha/issues/2546